internal/archive/fsops_test.go

```diff
@@ -111,9 +115,43 @@ var _ = Describe("FSOPS", func() {
 			Entry("Target is empty string",
 				getFullPath("testdata", "mtahtml5"), "", nil, true, nil),
 			Entry("Source is empty string", "", "", nil, true, nil),
+			// folder module (which is symbolic link itself) is archived
+			// it points to folder moduleNew that consists of symlink pointing to content and package.json
+			// etc... thus we check a complex case consisting of normal files/folders and symbolic links that are also
+			// files and folders
+			Entry("symbolic links",
+				getFullPath("testdata", "testsymlink", "module_symlink_dir"), targetFilePath, nil, false,
+				[]string{"module_symlink_dir/", "module_symlink_dir/symlink_dir/", "module_symlink_dir/package.json",
+					"module_symlink_dir/symlink_dir/test_dir/", "module_symlink_dir/symlink_dir/test_dir/test1.txt",
+					"module_symlink_dir/symlink_dir/test.txt",
+					"module_symlink_dir/symlink_dir/symlink2_dir/", "module_symlink_dir/symlink_dir/symlink2_dir/test3.txt",
+					"module_symlink_dir/symlink_dir/symlink2_dir/symlink3.txt"}),
 		)
 	})
 
+	var _ = Describe("addSymbolicLinkToArchive - failures", func() {
```

Comment:
Please add a test case for recursive symlinks after you handle it in the code

Reply:
done